### DISCUSSION TITLE
Where can I get the updated Pro version?

### DISCUSSION THREAD

**Discussion by jjwwczy** [omkarcloud/google-maps-scraper#142] [Q&A] [answered]
I can't find the sponsored page, so is there any way to get the updated pro version?

**Comment by Chetan11-dev** [comment]
You will have received an email with Pro Link on your github email, if you visit it you may download the latest updated version. Then you can follow https://github.com/omkarcloud/google-maps-scraper/blob/master/advanced.md#-how-to-update-the-scraper-to-the-latest-version to update it.

**Reply by jjwwczy** [reply]
I didn't set my email available before, now I have made it visible, could you please send the link to me?

**Reply by Chetan11-dev** [reply]
From which account did you sponsor?

**Reply by jjwwczy** [reply]
Exactly this account: jjwwczy. Maybe you can see my sponsorship, I sponsored "omkarcloud" for  [google-maps-scraper](https://github.com/omkarcloud/google-maps-scraper)

**Reply by Chetan11-dev** [reply, marked as answer]
I mailed the link to your university email.
Thanks :)

**Reply by jjwwczy** [reply]
Received. Thank you so much!

**Reply by 2GTaK** [reply]
@Chetan11-dev 
Can you send me the updated version as well?
Thank you. I am a sponsor but the download page is nowhere to be found.

**Reply by Chetan11-dev** [reply]
@2GTaK Could you mail me at chetan@omkar.cloud?
